### ISSUE TITLE
Expression parser invalid syntax doesn't throw

### ISSUE THREAD

**Issue by bigopon** [aurelia/aurelia#808]
# 🐛 Bug Report

Expression parser does not throw on invalid expression

## 🤔 Expected Behavior
For template:
```html
    <div repeat.for="entry of table">${entry.hour}:${entry:minute}</div>
```
An error would be thrown

## 😯 Current Behavior

It still works, but it treat the template as if it's only `${entry}`

## 💁 Possible Solution

N/A

## 💻 Code Sample

N/A

**Comment by Vheissu** [comment]
@bigopon Is this still an issue or can this be closed?

**Comment by bigopon** [comment]
resolved by #1549
